shared/src/packages/new-project-create/libs/validation-schemas/project-name.validation-schema.ts

```diff
@@ -0,0 +1,20 @@
+import joi from 'joi';
+
+import { ProjectNameValidationMessage } from '../enums/enums.js';
+
+const ProjectNameValidationSchema = joi.object({
+  projectName: joi
+    .string()
+    .min(1)
+    .max(30)
+    .regex(/^[\w!"#$%&'()*+,./:;<=>?@[\\\]^{|}~-]+$/)
+    .messages({
+      'string.name': ProjectNameValidationMessage.PROJECT_NAME_WRONG,
```

Comment:
it isn't a ` ProjectName`. it's a part of `site` validation

Reply:
i need to change path for all files from new-project-create to site, and change name like "SiteName" or something like this?